src/go/rpk/pkg/cli/cluster/health.go

```diff
@@ -72,13 +72,13 @@ following conditions are met:
 func printHealthOverview(hov *adminapi.ClusterHealthOverview) {
 	types.Sort(hov)
 	out.Section("CLUSTER HEALTH OVERVIEW")
-	overviewFormat := `Healthy:                     %v
-Unhealthy reasons:           %v
-Controller ID:               %v
-All nodes:                   %v
-Nodes down:                  %v
-Leaderless partitions:       %v
-Under-replicated partitions: %v
-`
-	fmt.Printf(overviewFormat, hov.IsHealthy, hov.UnhealthyReasons, hov.ControllerID, hov.AllNodes, hov.NodesDown, hov.LeaderlessPartitions, hov.UnderReplicatedPartitions)
+	tw := out.NewTable()
+	defer tw.Flush()
+	tw.Print("Healthy:", hov.IsHealthy)
+	tw.Print("Unhealthy reasons:", hov.UnhealthyReasons)
+	tw.Print("Controller ID:", hov.ControllerID)
+	tw.Print("All nodes:", hov.AllNodes)
+	tw.Print("Nodes down:", hov.NodesDown)
+	tw.Print(fmt.Sprintf("Leaderless partitions (%v):", hov.LeaderlessCount), hov.LeaderlessPartitions)
+	tw.Print(fmt.Sprintf("Under-replicated partitions (%v):", hov.UnderReplicatedCount), hov.UnderReplicatedPartitions)
```

Comment:
Changed to tab writer since now the width of the fields descriptions may vary depending on the count of the URP / leaderless partitions:

```
Leaderless partitions (1)

vs

Leaderless partitions (111)
```